fec/models.py

```diff
@@ -26,6 +26,19 @@ def party_abbreviation(self):
     def party_full(self):
         return self.PARTIES.get(self.party, self.party)
 
+    def similar_campaigns(self):
+        shared_contributors = Count(
+            'committee__contribution__contributor',
+            distinct=True,
+            filter=Q(
+                committee__contribution__contributor__contribution__committee__campaign__id
+                =self.id))
```

Comment:
Query to get the number of unique contributors for a campaign, who also contributed to the given campaign